### PR TITLE
Add support for the muliple widgets for restapi

### PR DESCRIPTION
**Pull request by rwx788**
https://github.com/libyui/libyui-rest-api/pull/3

### CODE REVIEW

File: package/libyui-ncurses.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Nov 29 14:41:35 UTC 2019 - Rodion Iafarov <riafarov@suse.com>
+
+- Add support to operate on many widgets with rest-api
```
Comment (shundhammer) [inline package/libyui-ncurses.changes]: Please mention the so number bump to 11.

File: package/libyui-ncurses.changes [hunk 2]
```diff
@@ -94,7 +101,7 @@ Mon Aug  7 13:21:35 UTC 2017 - mvidner@suse.com
 Tue Jul  4 07:58:10 UTC 2017 - mfilka@suse.com
 
 - bnc#1047145
-  - patch to make the package buildable by gcc7 (by werner@suse.com) 
+  - patch to make the package buildable by gcc7 (by werner@suse.com)
```
Comment (shundhammer) [inline package/libyui-ncurses.changes]: Your editor killed all the old trailing whitespace. I like that; but I am not sure if the OBS team will also like the fact that the diff is so large.

**Comment by shundhammer** [comment]
Just the two minor things, otherwise LGTM.

File: package/libyui-ncurses.changes (hunk 2 above)
Comment (rwx788) [reply package/libyui-ncurses.changes]: Yeah, I've decided to leave it. See no reason in drugging these forever.

File: package/libyui-ncurses.changes (hunk 1 above)
Comment (rwx788) [reply package/libyui-ncurses.changes]: Added, thanks!

**Comment by shundhammer** [comment]
The IBS changelog checker script will want a bsc# in the .changes file.

**Pull request merged**